script.module.enum34/lib/enum/setup.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
remove

Reply:
The code changes you are suggesting are from the original author which is now part of the Python 3.4+ distribution. Are you sure these proposed changes will not break the enum module?

Reply:
no this is only needed to packages for pypi